nuxt.config.js

```diff
@@ -3,7 +3,16 @@ export default defineNuxtConfig({
     '@nuxt/content'
   ],
   content: {
-    documentDriven: true
+    documentDriven: true,
+    markdown: {
+      remarkPlugins: {
+        'remark-sectionize': {},
```

Comment:
This wraps each heading and the content that follows it in a <section> tag. This is necessary to determine when a section is visible on the page, particularly for a long section where the header is no longer visible.

Reply:
In a section tag?

Reply:
Haha yes a section tag - the formatting seems to have actually applied a section tag 😂